Sources/App/Core/RSS.swift

```diff
@@ -62,10 +62,20 @@ extension RecentPackage {
             .title(packageName),
             .link(link),
             .pubDate(createdAt, timeZone: .utc),
-            .content(
-                .h2(.a(.href(link), .text(packageName))),
+            .description(
+                .p(
+                    .a(
+                        .href(link),
+                        .text(packageName)
+                    )
+                ),
                 .p(.text(packageSummary ?? "")),
-                .element(named: "small", nodes: [.a(.href(link), .text(packageName))])
+                .element(named: "small", nodes: [
+                    .a(
+                        .href(link),
+                        .text("\(repositoryOwner)/\(repositoryName)")
```

Comment:
Is this the lack of a leading `/` intentional? Since this is a link I think we should use one of

```
SiteURL.package(.value(repositoryOwner), .value(repositoryName)).absoluteURL()
SiteURL.package(.value(repositoryOwner), .value(repositoryName)).relativeURL()
SiteURL.package(.value(repositoryOwner), .value(repositoryName)).path
```

so in case of the above it'd be `.path`.

Reply:
Oh wait, this is the text part - never mind

Reply:
Yep, intentional. It was supposed to look like the [owner]/[repo] that people might recognise from GitHub, rather than being a URL.

Like this:

<img width="469" alt="Screenshot 2020-06-10 at 09 50 34@2x" src="https://user-images.githubusercontent.com/5180/84247390-f4489000-aaff-11ea-8751-7f676bc28ec1.png">